CONTRIBUTING.md

```diff
@@ -0,0 +1,79 @@
+# Contributing to Arbor
+
+First off, thank you for considering contributing to Arbor! It's people like you that
+make Arbor such a great tool. Feel welcome and read the following sections in order to
+know how to ask questions and how to work on something.
+
+## Types of contributions
+
+There are many ways to contribute, from writing tutorials or blog posts, improving the
+documentation, submitting bug reports and feature requests or writing code which can be
+incorporated into Arbor itself.
+
+If you come across a bug in Arbor, please [file an issue on the GitHub issue tracker](https://github.com/arbor-sim/arbor/issues/new).
+You can attach files to issues. Data and bits of code that help to reproduce the issue
+are very welcome; they'll make it easier for Arbor developers to debug and fix your issue.
+
+The documentation is found in the `/doc` subdirectory, C++ examples in `/example` and
+Python examples in `/python/example`. You can add yours in the same way you would contribute
+code, please see the "Github workflow" section.
+
+Please don't use the issue tracker for support questions. You can use the [Arbor Discussions](https://github.com/arbor-sim/arbor/discussions)
+page for this.
+
+## Github workflow
+
+Arbor uses Git for version control and Github to host its code repository and nearly
+all of its infrastructure.
+
+* If you're not familiar with Git or Github, please have at look at
+[this introduction](https://docs.github.com/en/free-pro-team@latest/github/getting-started-with-github/set-up-git).
+* Make sure you have a [GitHub account](https://github.com/signup/free).
+* If your problem or feature request is not yet filed in the issue tracker,
+please [file an issue](https://github.com/arbor-sim/arbor/issues) describing the problem,
+bug or feature. You can add supporting data, code or documents to help make your point.
+* If you want to write code that implements a solution or feature, the workflow is as follows:
+    1. Fork the Arbor repository
+    2. Create a feature branch off of master and implement your feature
```

Comment:
Could remove 'feature', reads clearer, since it's not always features. Also: less jargon.

Reply:
Done!